src/util.js

```diff
@@ -311,3 +311,16 @@ function setLocEnd(node, index) {
   }
 }
 util.setLocEnd = setLocEnd;
+
+// http://stackoverflow.com/a/7124052
+function htmlEscapeInsideDoubleQuote(str) {
+  return str
+    .replace(/&/g, '&amp;')
```

Comment:
Why are we doing this in the formatter? Won't this continually replace the `&` in `&amp;` if you successively print a file multiple times?